_posts/2020-10-10-the-importance-of-time-tracking.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "The importance of proper time tracking"
+date: 2020-10-10 15:00:00
+categories: ["team", "remote"]
+author: "ignaciobertotti"
+---
+
+As a software consulting company, we bill for the time we work on client projects. So it is vital for us to keep track of the time we put into every task.
+We've had some trouble over the years with our time tracking that led us to work more time than what we finally billed for. We would like to share some of the experience we have collected over the years and the tools that we currently use.
+
+<!--more-->
+
+## Implications of poor time tracking
+Poor time tracking can cause direct and indirect damage to this kind of companies.
+
+### Direct
+- Impact in revenue: Less hours tracked for client projects means less hours billed at the end of the billing period for the same amount of work. Over delivering is a great practice, but only if it is done on purpose. If not, it may go unnoticed.
+
+- Miscalculation of internal project costs: This cost is proportional to the time invested on this projects, so time tracking is vital on this subject.
+
+### Indirect
+- Time estimations for billable projects may turn unreliable and hourly rates can become affected as a result.
+
+- People tending to track less time than what they actually worked reflects directly in the company's efficiency, making it look higher than what it actually is.
+
+- Internal processes, like calls, may be taking more time than necessary. Properly tracking the time spent on these calls is the best way of troubleshooting internal efficiency issues.
+
+## Improving Time Tracking Skills
```

Comment:
Stay consistent...up above you are not using all capitals in the titles. Choose one or the other style.

Reply:
Got it! Thanks!